dts/bindings/timer/st,stm32-lptim.yaml

```diff
@@ -10,3 +10,20 @@ include:
     property-blocklist:
         - st,prescaler
         - st,countermode
+
+properties:
+    st,prescaler:
+      type: int
+      required: false
+      default: 1
+      description: |
+        Clock divider at the input of the lptimer clock, default reset value is 'not divided'.
+      enum:
```

Comment:
Please add a comment on the consequences on the tick duration/precision and the fact that `SYS_CLOCK_TICKS_PER_SEC` may require to be tuned for high values (starting from ?).
Also, please provide some example of known working `st,prescaler`/`CONFIG_SYS_CLOCK_TICKS_PER_SEC` combination.
And maybe a indication stating that above X value there is no guarantee to achieve a correct kernel scheduling.

Reply:
give details in the description

Reply:
Let's try a more formal version (please correct if required)
```
        Clock divider at the input of the LPTIM clock, default reset value is 'not divided'.

        Prescaler allows to achieve higher LPTIM timeout and consequently higher core sleep durations,
        but impacts the tick precision. To be used with caution.
        CONFIG_SYS_CLOCK_TICKS_PER_SEC could be used to tune tick duration and gain precision,
        at kernel timing granularity cost.

        For example, when LPTIM is clocked by the LSE (32768Hz) and st,prescaler = <32>:
        LPTIM global timeout is 64 seconds with an increment of 0.97 ms.
        Using CONFIG_SYS_CLOCK_TICKS_PER_SEC = 4096, tick = 0.24 ms, LPTIM precision is 4 ticks.
        Using CONFIG_SYS_CLOCK_TICKS_PER_SEC = 1024, tick = 0.97 ms, LPTIM precision is 1 ticks.
```

Reply:
ok, description refined